conf/nginx.conf

```diff
@@ -30,17 +30,62 @@ server {
 	# Add index.php to the list if you are using PHP
 	index index.html index.htm index.nginx-debian.html;
 
-	location ~* \.(json|csv)$ {
-		add_header Access-Control-Allow-Origin *;
+	# Warning: headers added by add_header are overriden by
+	# other add_header directives at the same level.
+	#location ~* \.(json|csv)$ {
+	#	add_header Access-Control-Allow-Origin *;
+	#}
+
+	location /data/ {
```

Comment:
this will apply not only to `static.openfoodfacts.net/data` but also `world.openfoodfacts.net/data`, should we restrain it to only `static.` prefix ?

Reply:
we have used static and world subdomains for /data/ in the past, so we should keep it.